### ISSUE TITLE
Optimization weird finding.

### ISSUE THREAD

**Issue by Alamer1** [nicehash/NiceHashQuickMiner#268]
Hello again,

After building the rig at that time I installed the latest RC quick miner which was 0.4.2.1 and it worked fine but I noticed aggressive fan profiles "all I did was set it to medium optimization" updated to 0.4.3.0 and it was the same, updated today to 0.4.3.1 and I faced an issue that I reported in another post on Github, which made me redownload 0.4.3.0 and use that instead, uninstalled the quick miner I had installed, started the miner and hit medium and to my surprise, the fan speeds are timid than before, not going to crazy high limits and the temperature is better, though the power limit is exactly the same as before, probably my installation was bad? no idea, thought to report it just in case.

Thank you 
Amer

**Comment by nicehashdev** [comment]
Please, enable full detailed logging and attach the logs.

**Comment by Alamer1** [comment]
[exported_logs.zip](https://github.com/nicehash/NiceHashQuickMiner/files/6207921/exported_logs.zip)
here you go, still as i said in my previous message, i am back to using the old version now (0.4.3.0)

**Comment by nicehashdev** [comment]
If you let it be for a while, fans do not go down to some normal level?

**Comment by Alamer1** [comment]
that is correct, it keeps the fan speed at high even though the temperatures are reached, thats with the latest nicehash quick miner, right now its normal with the previous nicehash quick miner

**Comment by nicehashdev** [comment]
Please, provide me a screenshot of OCTune when this happens.

**Comment by Alamer1** [comment]
i do believe that its the installation before, because i reupdated to the latest RC now and i am not getting the same issue anymore as when i first installed the RC, it is pretty normal now

**Comment by nicehashdev** [comment]
Okay, closing, considering fixed.
